.travis.yml

```diff
@@ -19,6 +19,8 @@ matrix:
 
 addons:
   apt:
+    sources:
+      - ubuntu-toolchain-r-test
```

Comment:
I've removed it. That repo was needed to get qt5 packages on ubuntu 14.04 at some point in history I think, not anymore.